Earthfile

```diff
@@ -467,23 +467,39 @@ base-image:
 
     RUN rm -rf /tmp/*
 
-    DO +CONTAINER_IMAGE_VERSION -VERSION=${OS_VERSION}
-    ARG _CIMG=$(cat IMAGE)
+    COPY ./naming.sh .
+    RUN ./naming.sh container_artifact_name
+    RUN ./naming.sh container_artifact_name > ./IMAGE
```

Comment:
should we remove naming.sh from the base image?

Reply:
Good point. I'll do.

Reply:
Done (right below)